README.md

```diff
@@ -65,18 +65,19 @@ you may need to run this command with sudo.
 
 #### Contributing
 
-We love code contributions. See the contributing.txt file for
+We love code contributions. See the [contributing.md](contributing.md) file for
 details.
 
 
 #### IRC
 
-The irc channel for Meson is `#mesonbuild` over at Freenode.
+The irc channel for Meson is `#mesonbuild` over at 
+[Freenode](https://webchat.freenode.net/#mesonbuild).
```

Comment:
I am not sure if we should link to the webchat implicitly like that. Maybe add a separate sentence about it?

Reply:
How about `You can use [FreeNode's official webchat](https://webchat.freenode.net/#mesonbuild) to [...]`?